src/governance.cpp

```diff
@@ -281,6 +281,9 @@ void CGovernanceManager::ProcessMessage(CNode* pfrom, std::string& strCommand, C
     // A NEW GOVERNANCE OBJECT VOTE HAS ARRIVED
     else if (strCommand == NetMsgType::MNGOVERNANCEOBJECTVOTE)
     {
+        // Ignore such messages until masternode list is synced
+        if(!masternodeSync.IsMasternodeListSynced()) return;
+
```

Comment:
The idea is that having mn list non-synced will lead to "unknown masternode" quite often which is not really helpful while you are still syncing imo so it's better to wait.